.../lib/ripstream_mp3.c:638:5: error: unknown type name '__uint32_t'; did you mean 'uint32_t'?
  638 |     __uint32_t framesize = 0;
      |     ^~~~~~~~~~
      |     uint32_t

---

--- a/lib/ripstream_mp3.c
+++ b/lib/ripstream_mp3.c
@@ -631,11 +631,11 @@ write_id3v2_frame (RIP_MANAGER_INFO *rmi
     int rc;
     char bigbuf[HEADER_SIZE] = "";
     ID3V2frame id3v2frame;
-#ifndef WIN32
+#if !defined(WIN32) && defined(__GLIBC__)
     __uint32_t framesize = 0;
 #else
     unsigned long int framesize = 0;
-#endif
+#endif /* musl */
 
     memset(&id3v2frame, '\000', sizeof(id3v2frame));
     strncpy(id3v2frame.id, tag_name, 4);
